File: lalsimulation/swig/python/check-swiglalsimulation.py
# check SWIG Python module wrapping the LALSimulation library
# Author: Karl Wette, 2011

import os

def msg(str):
    print(os.path.basename(__file__) + ": " + str)

# check module load
import swiglalsimulation
from swiglalsimulation import *
msg("passed module load")

# passed all tests!
msg("================")
msg("PASSED all tests")
msg("================")

File: lalsimulation/swig/octave/check-swiglalsimulation.m
## check SWIG Octave module wrapping the LALSimulation library
## Author: Karl Wette, 2011

function msg(str)
  disp([program_name, ": ", str]);
endfunction

## check module load
swiglalsimulation;
msg("passed module load");

## passed all tests!
msg("================");
msg("PASSED all tests");
msg("================");
